utilities/backupable/backupable_db.cc

```diff
@@ -216,7 +216,7 @@ class BackupEngineImpl : public BackupEngine {
     ~BackupMeta() {}
 
     void RecordTimestamp() {
-      env_->GetCurrentTime(&timestamp_);
+      env_->GetCurrentTime(&timestamp_).PermitUncheckedError();
```

Comment:
Perhaps better to return the Status from this function? It appears to only be used in one place, which itself is inside a function that can return Status.

Reply:
I would rather not.  The caller of that code is pretty complicated and probably over-engineered.  Most likely a lot of the pre-conditions should be refactored for Status and prevent work items from being created to begin with.  But this seems like a separate engineering effort from the Status checks.